model_prices_and_context_window.json

```diff
@@ -7566,30 +7566,6 @@
         "litellm_provider": "cloudflare", 
         "mode": "chat"
       },
-      "voyage/voyage-01": {
-        "max_tokens": 4096,
-        "max_input_tokens": 4096,
-        "input_cost_per_token": 0.0000001,
-        "output_cost_per_token": 0.000000,
-        "litellm_provider": "voyage",
-        "mode": "embedding"
-    },
-    "voyage/voyage-lite-01": {
```

Comment:
this model is still live, just deprecated - https://docs.voyageai.com/docs/embeddings